src/components/Event/Event.st.css

```diff
@@ -0,0 +1,143 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback, font;
+}
+
+:import {
+    -st-from: "../Text/Text.st.css";
+    -st-default: TPAText;
+}
+
+/*Defaults*/
+:vars {
+    DefaultFontStyle: "{theme: 'Body-M', size: '14px', lineHeight: '1em'}";
+    DefaultTimeColor: color-4;
+    DefaultTitleColor: color-5;
+    DefaultAllDayBackground: color-8;
+    DefaultAllDayOpacity: 0.2; 
+    DefaultTopPadding: 4px;
+    DefaultRightPadding: 4px;
+    DefaultBottomPadding: 4px;
+    DefaultLeftPadding: 12px;
+    DefaultRTLTopPadding: 4px;
+    DefaultRTLRightPadding: 12px;
+    DefaultRTLBottomPadding: 4px;
+    DefaultRTLLeftPadding: 4px;
+    BeforeColor: color(color-4);
+    BeforeOpacity: 0.06;
+}
+
+/*Overrides*/
+:vars {
+    FontStyle: --overridable;
+    TimeColor: --overridable;
+    TitleColor: --overridable;
+    AllDayBackground:  --overridable;
+    AllDayOpacity: --overridable;
+    TopPadding: --overridable;
+    RightPadding: --overridable;
+    BottomPadding: --overridable;
+    LeftPadding: --overridable;
+    RTLTopPadding: --overridable;
+    RTLRightPadding: --overridable;
+    RTLBottomPadding: --overridable;
+    RTLLeftPadding: --overridable;
+}
+
+/* Fallbacks */
+:vars {
+    CurrentFontStyle: fallback(value(FontStyle), font(value(DefaultFontStyle)));
+    CurrentTimeColor: color(fallback(value(TimeColor), value(DefaultTimeColor)));
+    CurrentTitleColor: color(fallback(value(TitleColor), value(DefaultTitleColor)));
+    CurrentAllDayBackground: color(fallback(value(AllDayBackground), value(DefaultAllDayBackground)));
+    CurrentAllDayOpacity: fallback(value(AllDayOpacity), value(DefaultAllDayOpacity));
+    CurrentTopPadding: fallback(value(TopPadding),value(DefaultTopPadding));
+    CurrentRightPadding: fallback(value(RightPadding),value(DefaultRightPadding));
+    CurrentLeftPadding: fallback(value(LeftPadding),value(DefaultLeftPadding));
+    CurrentBottomPadding: fallback(value(BottomPadding),value(DefaultBottomPadding));
+    CurrentRTLTopPadding: fallback(value(RTLTopPadding),value(DefaultRTLTopPadding));
+    CurrentRTLRightPadding: fallback(value(RTLRightPadding),value(DefaultRTLRightPadding));
+    CurrentRTLLeftPadding: fallback(value(RTLLeftPadding),value(DefaultRTLLeftPadding));
+    CurrentRTLBottomPadding: fallback(value(RTLBottomPadding),value(DefaultRTLBottomPadding));
```

Comment:
did you mean:
```suggestion
    DefaultFontStyle: fallback(value(FontStyle), font(value(DefaultFontStyle)));
    DefaultTimeColor: color(fallback(value(TimeColor), value(DefaultTimeColor)));
    DefaultTitleColor: color(fallback(value(TitleColor), value(DefaultTitleColor)));
    DefaultAllDayBackground: color(fallback(value(AllDayBackground), value(DefaultAllDayBackground)));
    DefaultAllDayOpacity: fallback(value(AllDayOpacity), value(DefaultAllDayOpacity));
    DefaultTopPadding: fallback(value(TopPadding),value(DefaultTopPadding));
    DefaultRightPadding: fallback(value(RightPadding),value(DefaultRightPadding));
    DefaultLeftPadding: fallback(value(LeftPadding),value(DefaultLeftPadding));
    DefaultBottomPadding: fallback(value(BottomPadding),value(DefaultBottomPadding));
    DefaultRTLTopPadding: fallback(value(RTLTopPadding),value(DefaultRTLTopPadding));
    DefaultRTLRightPadding: fallback(value(RTLRightPadding),value(DefaultRTLRightPadding));
    DefaultRTLLeftPadding: fallback(value(RTLLeftPadding),value(DefaultRTLLeftPadding));
    DefaultRTLBottomPadding: fallback(value(RTLBottomPadding),value(DefaultRTLBottomPadding));
```

Reply:
I meant the first one. the Current gets the fallback: if we didn't get the override for the style then we use the default. why ?

Reply:
I meant that usually our fallback variables are called `Default...` and not `Current...`

Reply:
and do we really need all these paddings?

Reply:
I followed the checkbox for conventions. I don't mind change it, just tell me to do so.
About the paddings I'll fix it.